app/src/androidTest/java/com/duckduckgo/app/di/StubAppConfigurationDownloadModule.kt

```diff
@@ -17,11 +17,17 @@
 package com.duckduckgo.app.di
 
 import com.duckduckgo.app.job.ConfigurationDownloader
+import com.duckduckgo.di.scopes.AppObjectGraph
+import com.squareup.anvil.annotations.ContributesTo
 import dagger.Module
 import dagger.Provides
 import io.reactivex.Completable
 
 @Module
+@ContributesTo(
+    scope = AppObjectGraph::class,
+    replaces = [AppConfigurationDownloaderModule::class]
```

Comment:
This is how we replace dagger modules for testing with Anvil. No more TestComponent needed